src/cljs/org/broadinstitute/firecloud_ui/page/status.cljs

```diff
@@ -34,8 +34,12 @@
            [:div {} "Response text:" [:br] (-> (:xhr response) .-responseText)]]))])
    :component-did-mount
    (fn [{:keys [props state]}]
-     (utils/ajax-orch (:path (:request props))
```

Comment:
That ends up putting nil into `:service-prefix` so that all of the other api calls no longer have `"/service/api"`